hack/verify-api-groups.sh

```diff
@@ -112,8 +112,8 @@ for expected_install_package in "${expected_install_packages[@]}"; do
 	fi
 
 	for known_version_file in "${known_version_files[@]}"; do
-		if ! grep -q "${expected_install_package}/install" ${known_version_files} ; then
-			echo "missing ${expected_install_package}/install from ${known_version_files}"
+		if ! grep -q "${expected_install_package}/install" "${known_version_file}" ; then
```

Comment:
this one seems kinda concerning. we were really just checking the first file, trying to figure out if the second entry should be removed or corrected in some way.
/hold